versions.json

```diff
@@ -266,7 +266,7 @@
         "fpm_directories": ["bin","include","lib"]
     },
     "mungefs": {
-        "commitish": "1.0.2",
+        "commitish": "bump",
```

Comment:
tweak

Reply:
Using 1.0.3 which will be released shortly.